pkg/ring/lifecycler_test.go

```diff
@@ -43,6 +43,8 @@ func checkNormalised(d interface{}, id string) bool {
 }
 
 func TestLifecycler_HealthyInstancesCount(t *testing.T) {
+	t.Parallel()
```

Comment:
Please ensure all changes in a commit are explained by the commit message.
Specifically for making tests run in parallel, I would need strong justification to do it at the same time as changing the behaviour of the code being tested.

Reply:
Was just to speed up tests. Not related to these changes (a part from that I noticed while adding new tests). Anyway, I removed `t.Parallel()` and I will propose it in a separate PR to keep discussion isolated.